src/mono/mono/mini/interp/interp.c

```diff
@@ -8900,4 +8901,52 @@ mono_jiterp_get_opcode_info (int opcode, int type)
 	}
 }
 
+EMSCRIPTEN_KEEPALIVE int
+mono_jiterp_dummy_trace (void *_frame, void *pLocals, JiterpreterCallInfo *cinfo, const guint16 *ip)
```

Comment:
consider renaming it. Is this the placeholder/stub/trampoline ?

Reply:
Yeah this is the placeholder. I'll come up with new names if this one isn't obvious.